daemon/src/projection.rs

```diff
@@ -627,6 +630,21 @@ impl From<Order> for CfdOrder {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, derive_more::Display)]
+pub struct Identity(String);
```

Comment:
I am not sure I follow your reasoning on the commit message. The `model::Identity` serializes exactly to what you need. What is the issue?